substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -254,7 +255,7 @@ pub mod pallet {
 		/// validators, they can chill at any point, and their approval stakes will still be
 		/// recorded. This implies that what comes out of iterating this list MIGHT NOT BE AN ACTIVE
 		/// VALIDATOR.
-		type TargetList: SortedListProvider<Self::AccountId, Score = BalanceOf<Self>>;
+		type TargetList: SortedListProvider<Self::AccountId, Score = VoteWeight>;
```

Comment:
I recall the approval stake of a validator could very well be above `VoteWeight`, is this a deliberate change?

Reply:
The doc for `EventListeners` is also now out of date.